cms/src/components/content/ContentOverview.vue

```diff
@@ -77,6 +79,44 @@ if (!Object.entries(TagType).some((t) => t[1] == tagTypeString)) tagTypeString =
 
 const titleType = tagTypeString ? tagTypeString : props.docType;
 router.currentRoute.value.meta.title = `${capitaliseFirstLetter(titleType)} overview`;
+
+const searchTerm = ref("");
+
+const sortOptionsAsRef = ref(null);
+
+const showSortOptions = ref(false);
+
+const selectedSortOption = ref("updatedTimeUtc");
+
+watch(searchTerm, () => {
+    setTimeout(() => {
+        queryOptions.value.search = searchTerm.value;
+    }, 500);
+});
```

Comment:
I don't think this works the way you think it will: What I think happens here is that for every update to `searchTerm`, it will create a new timeout, so in the end we're still updating `queryOptions` every time the user types. Instead, what we want to do is called a "debounce" or "debouncing the input". This can be achieved in multiple ways. The lodash library, which we already use, has `debounce` method.